2022-04-10 11:53:59,409 INFO    MainThread:91414 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-04-10 11:53:59,409 INFO    MainThread:91414 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/settings
2022-04-10 11:53:59,409 INFO    MainThread:91414 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-10 11:53:59,409 INFO    MainThread:91414 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-10 11:53:59,409 INFO    MainThread:91414 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220410_115359-1clau6hw/logs/debug.log
2022-04-10 11:53:59,409 INFO    MainThread:91414 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220410_115359-1clau6hw/logs/debug-internal.log
2022-04-10 11:53:59,410 INFO    MainThread:91414 [wandb_init.py:init():439] calling init triggers
2022-04-10 11:53:59,410 INFO    MainThread:91414 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-04-10 11:53:59,410 INFO    MainThread:91414 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 1pxb5egj
2022-04-10 11:53:59,411 INFO    MainThread:91414 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/1pxb5egj
2022-04-10 11:53:59,412 INFO    MainThread:91414 [jupyter.py:save_history():429] not saving jupyter history
2022-04-10 11:53:59,412 INFO    MainThread:91414 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-10 11:53:59,412 INFO    MainThread:91414 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-04-10 11:53:59,412 INFO    MainThread:91414 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-10 11:53:59,416 INFO    MainThread:91414 [wandb_run.py:_restore():1674] restore
2022-04-10 11:53:59,647 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 662821
  total_bytes: 662821
}

2022-04-10 11:53:59,826 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 662821
  total_bytes: 662821
}

2022-04-10 11:54:00,749 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 662821
  total_bytes: 668548
}

2022-04-10 11:54:00,851 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 662821
  total_bytes: 668548
}

2022-04-10 11:54:00,953 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 663747
  total_bytes: 668548
}

2022-04-10 11:54:01,056 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 668548
  total_bytes: 668548
}

2022-04-10 11:54:01,158 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 668548
  total_bytes: 668548
}

2022-04-10 11:54:01,260 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 668548
  total_bytes: 668548
}

2022-04-10 11:54:01,361 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 668548
  total_bytes: 668548
}

2022-04-10 11:54:01,463 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 668548
  total_bytes: 668548
}

2022-04-10 11:54:02,145 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 668548
  total_bytes: 668548
}

2022-04-10 11:54:02,570 INFO    MainThread:91414 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  media_count: 16
}
pusher_stats {
  uploaded_bytes: 668548
  total_bytes: 668548
}
local_info {
}

2022-04-10 11:54:03,747 INFO    MainThread:91414 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-10 11:54:03,748 INFO    MainThread:91414 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-10 11:54:03,755 INFO    MainThread:91414 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-04-10 11:54:03,762 INFO    MainThread:91414 [wandb_init.py:init():492] starting backend
2022-04-10 11:54:03,763 INFO    MainThread:91414 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-10 11:54:03,770 INFO    MainThread:91414 [backend.py:ensure_launched():219] starting backend process...
2022-04-10 11:54:03,879 INFO    MainThread:91414 [backend.py:ensure_launched():224] started backend process with pid: 91942
2022-04-10 11:54:03,881 INFO    MainThread:91414 [wandb_init.py:init():501] backend started and connected
2022-04-10 11:54:03,907 INFO    MainThread:91414 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-04-10 11:54:03,908 INFO    MainThread:91414 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-10 11:54:03,908 INFO    MainThread:91414 [wandb_init.py:init():565] updated telemetry
2022-04-10 11:54:04,073 INFO    MainThread:91414 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-10 11:54:05,779 INFO    MainThread:91414 [wandb_run.py:_on_init():1759] communicating current version
2022-04-10 11:54:05,842 INFO    MainThread:91414 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-10 11:54:05,842 INFO    MainThread:91414 [wandb_init.py:init():625] starting run threads in backend
2022-04-10 11:54:07,070 INFO    MainThread:91414 [wandb_run.py:_console_start():1733] atexit reg
2022-04-10 11:54:07,071 INFO    MainThread:91414 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-04-10 11:54:07,071 INFO    MainThread:91414 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-04-10 11:54:07,073 INFO    MainThread:91414 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-10 11:54:07,073 INFO    MainThread:91414 [wandb_init.py:init():664] run started, returning control to user process
